test/helpers/kubectl.go

```diff
@@ -3405,6 +3405,32 @@ func (kub *Kubectl) WaitForIPCacheEntry(node, ipAddr string) error {
 		&TimeoutConfig{Timeout: HelperTimeout})
 }
 
+// KeepRunning runs command on repeat in goroutine until channel
```

Comment:
Please adjust the commit message to explain the actual fix for the test instead of just mentioning the newly introduced function.